pandas/core/arrays/timedeltas.py

```diff
@@ -180,6 +180,9 @@ def _generate_range(cls, start, end, periods, freq, closed=None):
 
         return cls._simple_new(index, freq=freq)
 
+    # ----------------------------------------------------------------
```

Comment:
leftover? Will this cause merge conflicts with another branch you have going?

Reply:
The section is currently empty, put it in place for consistency.  I don't expect any conflicts ATM.